### PR TITLE
Open twitter on tapping tweets from list

### PR DESCRIPTION
**Pull request by prabal4546**
Fix to Issue #35

### CODE REVIEW

**Comment by mikaelacaron** [comment]
@prabal4546 Please resolve conflicts before this can be merged

**Comment by prabal4546** [comment]
These conflicts popped up due to the latest commits, anyways I think it's updated w.r.t to latest changes now👍🏻

File: brain-marks/Categories/Views/CategoryList.swift [hunk 1]
```diff
@@ -48,7 +48,55 @@ struct CategoryList: View {
...
+                            .foregroundColor(.white)
+                            .background(Color(UIColor(named: "twitter")!))
+                            .font(.system(size: 20, weight: .semibold, design: .default))
+                            .cornerRadius(10)
+                    }
+                }
+                .padding(20)
+            }
+            .navigationBarTitle("New Category")
+            .sheet(isPresented:$showAddURLView) {
```
Comment (mikaelacaron) [inline brain-marks/Categories/Views/CategoryList.swift:99]: All of this shouldn't be here. There is a separate file called NewCategorySheetView. It's located at: brain-marks/Categories/Views

Do you has this file?

**Comment by prabal4546** [comment]
Separated NewCategorySheetView

**Pull request closed without merging**